app/views/admin/badge_achievements/award.html.erb

```diff
@@ -1,23 +1,30 @@
-<h2 class="crayons-title mb-6">Award Badges</h2>
+<header class="flex justify-between items-center mb-4">
+  <h2 class="crayons-title">Award Badges</h2>
+  <%= link_to "Back to Badge Achievements", admin_badge_achievements_path, class: "c-link c-link--block" %>
```

Comment:
I'd be actually in favor of removing that link entirely because we are so inconsistent when it comes to showing these "back" links on other views.. i think we'll try to think about page header more holistically at some point, i.e. where goes title and how it supposed to look, where go tabs and how they supposed to look, where go breadcrumbs (and back links) and how they supposed to look, etc... but we're still pretty far away from this state (or maybe not..?) so i don't want to overthink these things now. **would you rather rename it somehow or remove it entirely for now?**

Reply:
I don't really have a preference either way - like you say, it was inconsistent before anyway 🤷‍♀️  Happy to defer to whatever you think is best 👍